core/src/main/java/tc/oc/pgm/api/map/Gamemode.java

```diff
@@ -1,31 +1,37 @@
 package tc.oc.pgm.api.map;
 
 public enum Gamemode {
-  ATTACK_DEFEND("ad"),
-  ARCADE("arcade"),
-  BLITZ("blitz"),
-  BLITZ_RAGE("br"),
-  CAPTURE_THE_FLAG("ctf"),
-  CONTROL_THE_POINT("cp"),
-  CAPTURE_THE_WOOL("ctw"),
-  DESTROY_THE_CORE("dtc"),
-  DESTROY_THE_MONUMENT("dtm"),
-  FREE_FOR_ALL("ffa"),
-  FLAG_FOOTBALL("ffb"),
-  KING_OF_THE_HILL("koth"),
-  KING_OF_THE_FLAG("kotf"),
-  MIXED("mixed"),
-  PAYLOAD("pd"),
-  RAGE("rage"),
-  RACE_FOR_WOOL("rfw"),
-  SCOREBOX("scorebox"),
-  SKYWARS("skywars"),
-  DEATHMATCH("tdm");
+  ATTACK_DEFEND("ad", "Attack/Defend", "A/D"),
+  ARCADE("arcade", "Arcade", "Arcade"),
+  BLITZ("blitz", "Blitz", "Blitz"),
+  BLITZ_RAGE("br", "Blitz: Rage", "Blitz: Rage"),
+  CAPTURE_THE_FLAG("ctf", "Capture The Flag", "CTF"),
+  CONTROL_THE_POINT("cp", "Control the Point", "CP"),
+  CAPTURE_THE_WOOL("ctw", "Capture the Wool", "CTW"),
+  DESTROY_THE_CORE("dtc", "Destoy the Core", "DTC"),
+  DESTROY_THE_MONUMENT("dtm", "Destroy the Monument", "DTM"),
+  FREE_FOR_ALL("ffa", "Free for All", "FFA"),
+  FLAG_FOOTBALL("ffb", "Flag Football", "FFB"),
+  KING_OF_THE_HILL("koth", "King of the Hill", "KotH"),
+  KING_OF_THE_FLAG("kotf", "King of the Flag", "KotF"),
+  MIXED("mixed", "Mixed", "Mixed"),
+  PAYLOAD("pd", "Payload", "PD"),
```

Comment:
Can we switch `Payload` from `pd` to `payload` pd isn't a great abbreviation (I know it isn't 100% you).